sdks/python/src/opik/integrations/bedrock/chunks_aggregator.py

```diff
@@ -35,3 +35,15 @@ def aggregate(items: List[Dict[str, Any]]) -> Dict[str, Any]:
                 result["metrics"]["latencyMs"] = metadata["metrics"]["latencyMs"]
 
     return result
+
+
+def aggregate_invoke_agent_chunks(items: List[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
let's rename existing `aggregate` function to `aggregate_converse_stream_chunks`.